content/tutorial/02-sveltekit/01-concepts/05-prerendering/README.md

```diff
@@ -2,4 +2,10 @@
 title: Prerendering
 ---
 
-TODO
+Prerendering means computing the contents of a page at build time and saving the HTML for display. You can opt in to prerendering on a per-page-basis or even the whole app when using SvelteKit.
+
+This approach has the same benefits as traditional server-rendered pages, but avoids recomputing the page for each visitor and so scales nearly for free as the number of visitors increases. The tradeoff is that the build process is more expensive and prerendered content can only be updated by building and deploying a new version of the application.
+
+Not all pages can be prerendered. The basic rule is this: for content to be prerenderable, any two users hitting it directly must get the same content from the server, and the page must not contain actions. Note that you can still prerender content that is loaded based on the page's parameters as long as all users will be seeing the same prerendered content.
+
+Pre-rendered pages are not limited to static content. You can build personalized pages if user-specific data is fetched and rendered client-side. This is subject to the caveat that you will experience the downsides of not doing SSR for that content as discussed above.
```

Comment:
```suggestion
You can add user-specific data to a prerendered page by fetching it from the browser instead of using SvelteKit's built-in data loading, though this is generally not recommended since it will cause content to pop in as it loads.
```